app/jobs/do_audit.rb

```diff
@@ -0,0 +1,5 @@
+require "audit_manager"
+
+# To run it
+# bin/rails runner app/jobs/do_audit.rb 
+AuditManager.system_audit()
```

Comment:
This file is unnecessary because `rails runner` works directly with ruby code. In this case:

```
bin/rails runner "AuditManager.system_audit"
```

This is why we chose to make it a class method rather than an instance method -- to keep it concise.

https://guides.rubyonrails.org/command_line.html#bin-rails-runner